.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{

```

Comment:
I don't agree with a `.vscode` config or `.editorconfig`. Maybe add Husky to prevent the error of lint, and add a check inside our CI. 
And add `.vscode/` to `.gitignore`

Reply:
I think adding husky is the best option. Do we have any CI set-up?

Reply:
If we choose to write a component project, similar to material-ui. We can use https://github.com/reakit/reakit as a example, since the main behaviour of this project is Focus on accessibility.

Reply:
> I think adding husky is the best option. Do we have any CI set-up?

If is just for lint and unit tests, I have some projects with circleCI: https://github.com/RafaelGSS/fastify-amqp/blob/master/.circleci/config.yml

Reply:
We need talk about the goal in #4. Anyway, if idea change, I will remove react and just leave the config files